### DISCUSSION TITLE
Unable to see where to edit the domain DNS Conofiguration

### DISCUSSION THREAD

**Discussion by donah123** [vercel/community#6925] [Help] [answered]
### Summary

I registered this domain on a registrar, i have pointed the DNS server to vercel, however, i couldn't find where to edit the DNS records

### Steps to Reproduce

Point the domain DNS Server to vercel

**Comment by ikku47** [comment]
same issue for me , did you find a solution ?

**Reply by donah123** [reply]
none yet

**Comment by swarnava** [comment, marked as answer]
If you bought the domain using Personal account, you should be able to locate it from Settings > Domains: https://vercel.com/account/domains
